### PR TITLE
docs: update product release issue template

### PR DESCRIPTION
**Pull request by ther3sa**
Removed checks for interoperability, data sovereignty and threat modeling because of changes in the process
## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by evegufy — CHANGES_REQUESTED** [review]
Looks good!

just one thing, it might make sense to wait for https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/865 to be merged to remove also the Styleguide point, which is also moved to a TRG

And just one small hint: the pull request title is very long, would maybe make sense to summerize to into something like `docs: update product release issue template`

**Review by FaGru3n — APPROVED** [review]
LGTM, looks a bit cleaner now

**Comment by ther3sa** [comment]
> Looks good!
> 
> just one thing, it might make sense to wait for [eclipse-tractusx/eclipse-tractusx.github.io#865](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/865) to be merged to remove also the Styleguide point, which is also moved to a TRG
> 
> And just one small hint: the pull request title is very long, would maybe make sense to summerize to into something like `docs: update product release issue template`

Hey Evelyn,
what do you mean by Styleguide point? :)

**Comment by matbmoser** [comment]
@ther3sa there is a new TRG about styleguide:  https://eclipse-tractusx.github.io/docs/release/trg-9/trg-9-01

Honestly, I could keep it out of the v24.08, and maybe ensure that is compliant in the v24.12.

**Comment by evegufy** [comment]
> > Looks good!
> > just one thing, it might make sense to wait for [eclipse-tractusx/eclipse-tractusx.github.io#865](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/865) to be merged to remove also the Styleguide point, which is also moved to a TRG
> > And just one small hint: the pull request title is very long, would maybe make sense to summerize to into something like `docs: update product release issue template`
> 
> Hey Evelyn, what do you mean by Styleguide point? :)

I think I was confusing it with the point `User Journey`, I suggest to either remove it or define who should check that in the future, as the role of the PO doesn't exist anymore

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (evegufy) [inline .github/ISSUE_TEMPLATE/3_product_release_checks.md:29]: Please add empty line for proper md format

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (evegufy) [inline .github/ISSUE_TEMPLATE/3_product_release_checks.md:45]: Please add empty line for proper md format

**Review by evegufy — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (ther3sa) [reply .github/ISSUE_TEMPLATE/3_product_release_checks.md:29]: done :)

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (ther3sa) [reply .github/ISSUE_TEMPLATE/3_product_release_checks.md:46]: done :)

**Comment by ther3sa** [comment]
> > > > Looks good!
> > > > just one thing, it might make sense to wait for [eclipse-tractusx/eclipse-tractusx.github.io#865](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/865) to be merged to remove also the Styleguide point, which is also moved to a TRG
> > > > And just one small hint: the pull request title is very long, would maybe make sense to summerize to into something like `docs: update product release issue template`
> > > 
> > > 
> > > Hey Evelyn, what do you mean by Styleguide point? :)
> > 
> > 
> > I think I was confusing it with the point `User Journey`, I suggest to either remove it or define who should check that in the future, as the role of the PO doesn't exist anymore
> 
> @ther3sa could you please provide feedback about the removal of the `User Journey` or if it's not planned to remove it, who is supposed to check that?

I am currently in the process of clarifying how we will deal with the user journey in future. I will provide an update on this as soon as I have a result. :)

**Comment by stephanbcbauer** [comment]
@ds-hzimmer , @ds-tthies , @ther3sa ass discussed ... @ther3sa and I adapted the content. Please have a look beforehand. Is something missing? If yes, please adapt.

We should check topics redundant. Some checks are already done via TRGs

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (evegufy) [inline .github/ISSUE_TEMPLATE/3_product_release_checks.md:88]: point is redundant to TRG 1

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (evegufy) [inline .github/ISSUE_TEMPLATE/3_product_release_checks.md:62]: moving the point with the md file didn't answer my question: https://github.com/eclipse-tractusx/sig-release/pull/813#issuecomment-2309873532

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (evegufy) [inline .github/ISSUE_TEMPLATE/3_product_release_checks.md:97]:
```suggestion
- [ ] Release notes/Changelogs are updated
```

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (evegufy) [inline .github/ISSUE_TEMPLATE/3_product_release_checks.md:94]: the name - which is referring to deployment - doesn't make sense to me considering bellows
```suggestion
## Release Documentation
```

**Review by evegufy — CHANGES_REQUESTED** [review]

**Comment by matbmoser** [comment]
@stephanbcbauer and @ther3sa 
Can we also add something like: Architecture Approval? Was not somthing that was mentioned in the last technical sync? 

Do we also want to include dependencies? Like - Updated dependency graph?

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (matbmoser) [inline .github/ISSUE_TEMPLATE/3_product_release_checks.md:23]: Is this really necessary? Shall we not say Contributors?

**Review by matbmoser — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (ther3sa) [reply .github/ISSUE_TEMPLATE/3_product_release_checks.md:62]: Hi Evelyn,
it seems this point slipped through on our side, and we apologize for the oversight. 
Since there are currently no user journeys in place, we have decided to remove this item. We expect that the new detailed test coverage will adequately address this area.

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (ther3sa) [reply .github/ISSUE_TEMPLATE/3_product_release_checks.md]: Done

File: .github/ISSUE_TEMPLATE/3_product_release_checks.md
Comment (ther3sa) [reply .github/ISSUE_TEMPLATE/3_product_release_checks.md]: done

**Comment by ther3sa** [comment]
> @stephanbcbauer and @ther3sa Can we also add something like: Architecture Approval? Was not somthing that was mentioned in the last technical sync?
> 
> Do we also want to include dependencies? Like - Updated dependency graph?

Hi @matbmoser ,
the architecture approval will be addressed in the [feature issue](https://github.com/eclipse-tractusx/sig-release/pull/571).

Could you please clarify what you mean by the dependency graph? I’m not entirely sure I understand your idea correctly.

**Comment by ther3sa** [comment]
@matbmoser @evegufy Could please provide feedback on a basis of the current state. :)

**Review by matbmoser — APPROVED** [review]
I would merge it ASAP in case there needs a update another PR shall be created.

LGTM

**Comment by ther3sa** [comment]
> I would merge it ASAP in case there needs a update another PR shall be created.
> 
> LGTM

Thank you, that works for me. 👍 
I think  we can wait for Evelyn's approval as well, and once confirmed, you/we can proceed with the merge—assuming no further changes are required. :)

**Review by evegufy — APPROVED** [review]
`Backward compatibility maintained (depending Major/ Minor)` is kind of a duplication to [TRG 5.11](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-11/) but I'm approving as I don't want to block the merge anymore and we should regularly review this template anyway to make sure that it brings value to the release process.

**Pull request merged**
